app/components/task/holder.hbs

```diff
@@ -1,28 +1,9 @@
 <div class="task-card">
   <Task-Details @task={{@task}} />
-  <div class="task-card__status">
-    {{#if (eq @task.status "active")}}
-      <select name="percentage-complete" id="percentCompleted" {{on "change" this.onPercentageChange}}>
-        <option value="">Percentage Completed</option>
-        {{#each this.availablePercentOptions as |percent| }}
-          <option value={{percent}}>{{percent}}</option>
-        {{/each}}
-      </select>
-    {{/if}}
-    <select name="task-status" id="status" {{on "change" this.onStatusChange}}>
-      <option value="">status</option>
-      {{#each this.availableStatusOptions as |status| }}
-          <option value={{status}}>{{status}}</option>
-      {{/each}}
-    </select>
-  </div>
-  <div class="task-card__btn">
-    <button type="button" {{on "click" (fn @onTaskUpdate @task.id)}}>Update Task</button>
-  </div>
   <div class="progress-bar-container">
     <div class="progress-bar">
-      <div class="bar percent-{{@task.percentCompleted}}">
+      <input id="input" type="range" name="percentCompleted" value="{{@task.percentCompleted}}" min="0" max="100" step="25" {{on "change" this.onPercentageChange}} {{on "change" (fn @onTaskUpdate @task.id)}}
+      list="status"/>
```

Comment:
What is the significance of `list="status"` here?

Reply:
Not required here, don't know why it was added